examples/next-js-simple/package.json

```diff
@@ -44,16 +44,12 @@
     ]
   },
   "dependencies": {
-    "@builder.io/react": "^2.0.1",
-    "@builder.io/widgets": "^1.2.22",
-    "@vercel/fetch": "^6.1.0",
-    "atob": "^2.1.2",
-    "jest": "^26.6.3",
-    "next": "^12.1.0",
-    "next-seo": "^4.11.0",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
-    "react-merge-refs": "^1.1.0"
+    "@builder.io/react": "^2.0.2-7",
+    "@builder.io/widgets": "^1.2.22-7",
```

Comment:
I'll release patch once this pr is reviewed and update the dependencies in this example